apps/demos/Demos/TreeList/AIColumn/jQuery/index.js

```diff
@@ -0,0 +1,223 @@
+$(() => {
+  const aiService = new AzureOpenAI({
+    dangerouslyAllowBrowser: true,
+    deployment,
+    endpoint,
+    apiVersion,
+    apiKey,
+  });
+
+  async function getAIResponse(messages, signal) {
+    const params = {
+      messages,
+      model: deployment,
+      max_tokens: 1000,
+      temperature: 0.7,
+    };
+    const response = await aiService.chat.completions.create(params, { signal });
+    const result = response.choices[0].message?.content;
+
+    return result;
+  }
+
+  const aiIntegration = new DevExpress.aiIntegration({
+    sendRequest({ prompt }) {
+      const controller = new AbortController();
+      const signal = controller.signal;
+
+      const aiPrompt = [
+        { role: 'system', content: prompt.system },
+        { role: 'user', content: prompt.user },
+      ];
+
+      const promise = getAIResponse(aiPrompt, signal);
+
+      const result = {
+        promise,
+        abort: () => {
+          controller.abort();
+        },
+      };
+
+      return result;
+    },
+  });
+
+  const popupContentTemplate = function (vehicle) {
+    const {
+      Source,
+      LicenseName,
+      Author,
+      Edits,
+    } = vehicle;
+    const sourceLink = `https://${Source}`;
+    return $('<div>').append(
+      $('<p>')
+        .append($('<b>').text('Image licensed under: '))
+        .append($('<span>').text(LicenseName)),
+      $('<p>')
+        .append($('<b>').text('Author: '))
+        .append($('<span>').text(Author)),
+      $('<p>')
+        .append($('<b>').text('Source link: '))
+        .append(
+          $('<a>', {
+            href: sourceLink,
+            target: '_blank',
+          })
+            .text(sourceLink),
+        ),
+      $('<p>')
+        .append($('<b>').text('Edits: '))
+        .append($('<span>').text(Edits)),
+    );
+  };
+
+  const popup = $('#popup').dxPopup({
```

Comment:
Should we explicitly set the showCloseButton option? By default, it has different values ​​in different themes.